apiRoutes/authenticate.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const router = express.Router()
+const jwt = require('jsonwebtoken')
+
+router.use(express.json())
+
+//Model
+const { user } = require('../models')
+
+//Login
+router.post("/", async (req, res)=>{
+    let username = req.body.username
+    let password = req.body.password
+
+    let findUser = await user.findOne({where:{ username: username}}).catch((err)=>{
+        console.log(err)
+        return res.sendStatus(400)
+    })
+
+    //Username not exist
+    if(findUser==null) return res.sendStatus(404)
+
+    let userData = findUser.dataValues
+    let userID = findUser.user_ID
+    let userPassword = userData.password
+
+    //Check password
+    if(userPassword == password){
+        let token = jwt.sign({ user_ID: userID}, "SECRET")
```

Comment:
Please make `.env` ([enviroment file](https://www.npmjs.com/package/dotenv)) to set `SECRET` key.
to make sure that we don't have to manual change a `SECRET` key later.